specification.markdown

```diff
@@ -313,35 +313,47 @@ The `name` provides the official human-readable name of the `EBML Element`. The
 
 The `name` attribute is REQUIRED.
 
-#### level
+#### path
 
-The `level` notes at what hierarchical depth or depths the `EBML Element` MUST occur if used within an `EBML Document`. The `Root Element` of an `EBML Document` is at Level 0 and the `Child Elements` that it contains are at Level 1. The `level` MUST be expressed as an integer. `EBML Elements` defined as `global` and `recursive` MAY occur at a level greater than or equal to the defined `level`.
+The path of the `EBML Element`. This path MUST be defined with the full hierarchy of `EBML Elements` separated with a `/`. The top `EBML Element` in the path hierarchy being the first in the value. The syntax of the `path` attribute is defined using this Extended Backus-Naur Form (EBNF) [@!ISO.14977.1996] notation:
 
-The `level` attribute is REQUIRED.
+```
+EBMLFullPath          ::= EBMLFullPath, [EBMLElementOccurence]
+EBMLReferencePath     ::= EBMLParentPath, EBMLElementPath
+EBMLParentPath        ::= (EBMLPathAtom)* | AllWildcard
+EBMLElementPath       ::= EBMLPathAtom | EBMLPathAtomRecursive
+EBMLPathAtom          ::= "/", EBMLAtomName
+EBMLPathAtomRecursive ::= "(", EBMLPathAtom, ")", "+"
+EBMLAtomName          ::= (EBMLNameChar)+ | AllWildcard | AtLeastOnce
+EBMLNameChar          ::= [A-Z] | [a-z] | [0-9] | "-" | "."
+AllWildcard           ::= "*"
+AtLeastOnce           ::= "+"
+EBMLElementOccurence  ::= "(", [MinOccurence], ",", [MaxOccurence] ")" 
+MinOccurence          ::= [0-9]+
+MaxOccurence          ::= [0-9]+ | "unbounded"
+```
 
-#### global
+The `"*"`, `"+"`, `"("`, `")"`, `"["` and `"]"` symbols MUST be interpreted as they are defined in the EBNF.
 
-A boolean to express if an `EBML Element` MUST occur at its defined level or may occur within any `Parent Element`.
+The `MinOccurence` represent the minimum number of occurrences of this `EBML Element` within its `Parent Element`. Each instance of the `Parent Element` MUST contain at least this many instances of this `EBML Element`. If the `EBML Element` has an empty `EBMLParentPath` then `MinOccurence` refers to constaints on the occurrence of the `EBML Element` within the `EBML Document`. The semantic meaning of `MinOccurence` within an `EBML Schema` is considered analogous to the meaning of `minOccurs` within an `XML Schema`. `EBML Elements` with `MinOccurence` set to "1" that also have a `default` value (see [default](#default)) declared are not REQUIRED to be stored but are REQUIRED to be interpreted, see [Note on the Use of default attributes to define Mandatory EBML Elements](#note-on-the-use-of-default-attributes-to-define-mandatory-ebml-elements). An `EBML Element` defined with a `MinOccurence` value greater than zero is called a `Mandatory EBML Element`.
 
-The `global` attribute is OPTIONAL. If the `global` attribute is not present then that `EBML Element` is to be considered not global.
+If `MinOccurence` is not present then that `EBML Element` is considered to have a `MinOccurence` value of 0.
```

Comment:
I'm not sure if I understand EBNF enough to understand, but does the definition above allow `MinOccurence` to not be present?